sheetScript.gs

```diff
@@ -176,37 +196,38 @@ function updatePlaylists(sheet) {
   }
 }
 
-function addVideosToPlaylist(playlistId, videoIds) {
-  var errorCount = 0;
+function addVideosToPlaylist(playlistId, videoIds, idx = 0, successCount = 0, errorCount = 0) {
   var totalVids = videoIds.length;
-  if (videoIds.length < 200) {
```

Comment:
Conditional check should stay as per https://github.com/Elijas/auto-youtube-subscription-playlist-2/pull/72#issuecomment-617977185